modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/MustacheScriptEngine.java

```diff
@@ -118,20 +122,25 @@ private class MustacheExecutableScript extends TemplateScript {
 
         @Override
         public String execute() {
-            final StringWriter writer = new StringWriter();
+            StringWriter writer = new SizeLimitingStringWriter(MUSTACHE_SIZE_LIMIT);
```

Comment:
Do we want to properly hook this into circuit breakers? Is that possible/sensible?

Reply:
Getting this to work in a circuit-breaker-way is not trivial